cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -619,10 +619,20 @@ For services, you can retrieve:
 
 For workflows, you can retrieve:
 
-- the information about the specific run currently running::
```

Comment:
I would leave this line in, as even though it is erroneous, as people may be used to it. We can remove it when the depreciation warning is removed.